hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CustomFeesValidator.java

```diff
@@ -159,29 +159,11 @@ public void validateForFeeScheduleUpdate(
                                 collectorId, fixedFee.denominatingTokenId(), tokenRelationStore, tokenStore);
                     }
                 }
-                case FRACTIONAL_FEE -> {
-                    // fractional fee can be only applied to fungible common tokens
-                    validateFractionalFeeForFeeScheduleUpdate(token, tokenRelationStore, collectorId, fee);
-                }
-                case ROYALTY_FEE -> {
-                    // royalty fee can be only applied to non-fungible unique tokens
-                    validateTrue(
-                            isNonFungibleUnique(tokenType), CUSTOM_ROYALTY_FEE_ONLY_ALLOWED_FOR_NON_FUNGIBLE_UNIQUE);
-                    final var royaltyFee = fee.royaltyFeeOrThrow();
-                    if (royaltyFee.hasFallbackFee()
-                            && royaltyFee.fallbackFeeOrThrow().hasDenominatingTokenId()) {
-                        final var tokenNum = royaltyFee
-                                .fallbackFeeOrThrow()
-                                .denominatingTokenId()
-                                .tokenNum();
-                        final var tokenId =
-                                TokenID.newBuilder().tokenNum(tokenNum).build();
-                        validateExplicitTokenDenomination(collectorId, tokenId, tokenRelationStore, tokenStore);
```

Comment:
Where did this logic go? Is it already contained in `validateFractionalFeeForFeeScheduleUpdate()`?

Reply:
It is in 
```
case ROYALTY_FEE -> // royalty fee can be only applied to non-fungible unique tokens
                validateRoyaltyFee(tokenType, fee, tokenRelationStore, tokenStore);
```